docs/gfm-alerts-to-admonitions/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  buildPythonPackage,
+  pytestCheckHook,
+  markdown-it-py,
+  lib,
+  setuptools,
+  pytest-regressions,
+}:
+buildPythonPackage {
+  pname = "gfm-alerts-to-admonitions";
+  version = "0.0";
+  format = "pyproject";
+
+  src = ./.;
```

Comment:
This probably doesn't really matter in nixvim, but this is frowned upon because changes to _anything_ in `./.` will trigger a rebuild, even stuff in _this file_ that'd normally just be metadata or passthru.

Reply:
Yeah, I saw how nixos-render-docs did it, but I did not want to bother with the complexity of filtering. I'm betting on us not touching this part very often

Reply:
Normally `./src` would be best practice, but as I said it shouldn't be too important for an internal package only used when we build our docs.

Reply:
To clarify, I wasn't talking about the `filter` function used with `lib.cleanSourceWith`.

I was referring to https://github.com/NixOS/nixpkgs/issues/301014:

> Derivations that use `src = ./.` will be rebuilt on any change to the `.nix` file that includes them, including formatting changes caused by `nixfmt-rfc-style` for [RFC 166](https://github.com/NixOS/rfcs/blob/master/rfcs/0166-nix-formatting.md?rgh-link-date=2024-04-02T18%3A12%3A28Z).